### ISSUE TITLE
Set Mobile Breakpoints - Site buttons

### ISSUE THREAD

**Issue by jonpekk** [codeforboston/boston-liquor-license-tracker#220]
**Requirements not already implemented**: 
- Buttons should be full width
- Text that wraps should make the button taller (ideally this wouldn't happen but would be a fallback to prevent text that bleeds over)
- Ensure Padding, margin, and other pixel sizing is correct

<img width="851" height="357" alt="Image" src="https://github.com/user-attachments/assets/90c1fb4c-d411-45d3-bf7f-b90c8b09cadd" />
